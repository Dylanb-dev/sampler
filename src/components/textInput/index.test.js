/* eslint-disable */

import React from 'react'
import renderer from 'react-test-renderer'

import TextInput from './'

test('TextInput renders correctly', () => {
  const props = { value: '', onChange: () => {} }

  const tree = renderer.create(<TextInput {...props} />).toJSON()

  expect(tree).toMatchSnapshot()
})
