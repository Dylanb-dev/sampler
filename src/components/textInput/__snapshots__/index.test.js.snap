// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextInput renders correctly 1`] = `
<input
  onChange={[Function]}
  style={
    Object {
      "border": "none",
      "borderRadius": "32px",
      "boxShadow": "0px 8px 15px rgba(0, 0, 0, 0.1)",
      "fontSize": "16px",
      "margin": "16px",
      "padding": "16px",
      "transition": "all 0.3s ease 0s",
      "width": "calc(100% - 64px)",
    }
  }
  value=""
/>
`;
